src/diffusers/modeling_utils.py

```diff
@@ -228,6 +228,10 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 The specific model version to use. It can be a branch name, a tag name, or a commit id, since we use a
                 git-based system for storing models and other artifacts on huggingface.co, so `revision` can be any
                 identifier allowed by git.
+            subfolder (`str`, *optional*, defaults to `""`):
+                In case the relevant files are located inside a subfolder of the model repo on huggingface.co, you can
```

Comment:
```suggestion
                In case the relevant files are located inside a subfolder of the model repo (either remote in huggingface.co or downloaded locally), you can
```